metadata/metadata.go

```diff
@@ -1892,6 +2256,59 @@ func (serv *MetadataServer) genericCreate(ctx context.Context, res Resource, ini
 		serv.Logger.Error(errors.WithStack(err))
 		return nil, err
 	}
+
+	triggers := []string{}
```

Comment:
All this logic can probably be in propogate change. Lets go through this after the rest is cleaned up